data/eddiejaoude/testimonials/krishstwt.json

```
@@ -1,5 +1,5 @@
-{
-  "title": "God of OSS",
-  "description": "As I like to say, Eddie is the god of open source and community. Eddie is the best and devrel and content creation 😍 Eddie's mentoring via their Twitter and Github is such a good addition to my life",
-  "date": "2023-01-10"
-}
+{
+  "title": "God of OSS",

```

[Humorous suggestion] @krishstwt, I  I think you should name @eddiejaoude the Dr. Strange of OSS since he's knowledgeable, can come into many spaces, win people over with his charisma and kindness, and is British lol! 😄 
```suggestion
  "title": "God of OSS",

```

---

Actually @CBID2, for that we are making @EdlienTheory's profile